src/ServiceGetActiveAlarmInfo.c

```diff
@@ -0,0 +1,182 @@
+//ServiceGetActiveAlarmInfo.c
+
+// SPDX-FileCopyrightText: 2023, Yaskawa America, Inc.
+// SPDX-FileCopyrightText: 2023, Delft University of Technology
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "MotoROS.h"
+
+rcl_service_t g_serviceGetActiveAlarmInfo;
+
+ServiceGetActiveAlarmInfo_Messages g_messages_GetActiveAlarmInfo;
+
+typedef motoros2_interfaces__srv__GetActiveAlarmInfo_Response GetActiveAlarmInfoResponse;
+
+
+static micro_ros_utilities_memory_rule_t mem_rules_response_[] =
+{
+    {"result_message", 50},
```

Comment:
Tbh I don't remember (it's been a while).

Easy to change of course, so if we can come up with a more appropriate max length, we'll use that.

Reply:
@ted-miller: any preferences for a default/initial length of the `result_message` field?

Longest `result_message` is 47 chars ([here](https://github.com/gavanderhoorn/motoros2_interfaces/blob/31b171e2fc04d8f8bd72c99b79430aeb130238c3/srv/GetActiveAlarmInfo.srv)), so I suggest we set this to 64.

Default value for max string capacity is 20, so that would not be sufficient.

Reply:
I have no preference. 64 is fine with me